cron/purge_cache.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ *
+ * phpBB Media Embed PlugIn extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2020 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\mediaembed\cron;
+
+use \phpbb\config\config;
+use \phpbb\mediaembed\cache\purge;
+
+/**
+ * Mediaembed cron task.
+ */
+class purge_cache extends \phpbb\cron\task\base
+{
+	/** @var config $config */
+	protected $config;
+
+	/** @var purge $mediaembed_cache */
+	protected $mediaembed_cache;
+
+	/**
+	 * Constructor
+	 *
+	 * @param config       $config           Config object
+	 * @param purge        $mediaembed_cache Purge cache object
```

Comment:
Based on renaming the cache class, this would be commented as `MediaEmbed cache object` instead